### ISSUE TITLE
Question on the resulting RVs and barycentric correction

### ISSUE THREAD

**Issue by JonasKemmer** [megbedell/wobble#49]
Hello,
as far as I can understand, the resulting values for the RVs do not include the barycentric correction.
Does this mean, they are just converted by: v_meas = c * z_meas?
My Question is: _How is the barycentric correction considered in wobble?_
In Bedell+(2019) you write:

> The apparent stellar RV, [...], is a sum of the star's actual velocity about its center of mass [..] and the projected motion of the Earth about the Solar System barycenter..

However, according to Wright and Eastman (2014) this is not valid:

> Note that the correction to z_meas is multiplicative, not additive. One should not, therefore, formulate the "barycentric correction" as the velocity that should be added or subtracted from z_meas to find z_true,  [...]. That is, the difference between z_meas and z_true includes a cross term of magnitude 3 m/s.

But as far as I understand from the example scripts, that is exactly what you do, isn't it?

Best regards,
Jonas Kemmer

**Comment by megbedell** [comment]
Yes, you're right! I don't expect that change to matter much for the sort of RV amplitudes we're looking at, but it's a good point and I will implement a `results.apply_bervs()` function to do the correction in a relativity-safe way.

**Comment by JonasKemmer** [comment]
Thank's for your response. So for now I can easily apply barycorrpy myself - however an implementation in the code itself is certainly an useful enhancement.
